tests/system/model_monitoring/test_model_monitoring.py

```diff
@@ -338,8 +336,9 @@ def test_basic_model_monitoring(self, with_sql_target: bool) -> None:
 
         endpoint = endpoints_list[0]
 
-        # ML-6594
-        assert not endpoint.status.feature_stats
+        if not with_sql_target:
+            # ML-6594
+            assert not endpoint.status.feature_stats
```

Comment:
according to https://github.com/mlrun/mlrun/pull/6007 It suppose to be excluded in v3ioKV & SQL no?

Reply:
yes it's still a bug in sql

Reply:
OK so as discussed reopen https://iguazio.atlassian.net/browse/ML-6594. 
FYI @gtopper